packages/azure-themes/src/azure/AzureThemeDark.ts

```diff
@@ -5,55 +5,89 @@ import { FontSizes } from './AzureType';
   errorText: DarkSemanticColors.text.error,
-  focusBorder: DarkSemanticColors.controlOutlines.accent,
+  focusBorder: '#4894fe',
+  iconButtonColor: '#0078D4',
+  iconButtonDisabledColor: '#A19F9D',
+  iconHoveredColor: '#106EBE',
+  iconPressedColor: 'rgb(108, 184, 246)',
   inputBackground: DarkSemanticColors.background,
   inputText: DarkSemanticColors.text.value,
   inputBorder: CommonSemanticColors.textControlOutline.rest,
   inputBorderHovered: CommonSemanticColors.textControlOutline.hover,
   inputPlaceholderText: DarkSemanticColors.text.placeholder,
-  link: DarkSemanticColors.text.hyperlink,
-  linkHovered: DarkSemanticColors.text.hyperlink,
+  link: '#0078D4',
+  linkHovered: '#004578',
   listBackground: DarkSemanticColors.background,
   listHeaderBackgroundPressed: DarkSemanticColors.item.hover,
   listItemBackgroundChecked: DarkSemanticColors.item.select,
   listItemBackgroundCheckedHovered: DarkSemanticColors.item.select,
   listItemBackgroundHovered: DarkSemanticColors.item.hover,
   listText: DarkSemanticColors.text.body,
-  menuItemBackgroundHovered: DarkSemanticColors.item.hover,
-  menuItemBackgroundPressed: DarkSemanticColors.item.select,
-  primaryButtonBackground: DarkSemanticColors.primaryButton.rest.background,
-  primaryButtonBackgroundDisabled: DarkSemanticColors.disabledButton.background,
-  primaryButtonBackgroundHovered: DarkSemanticColors.primaryButton.hover.background,
-  primaryButtonBackgroundPressed: DarkSemanticColors.primaryButton.pressed.background,
-  primaryButtonText: DarkSemanticColors.primaryButton.rest.text,
-  primaryButtonTextDisabled: DarkSemanticColors.disabledButton.text,
-  primaryButtonTextHovered: DarkSemanticColors.primaryButton.hover.text,
+  menuItemBackgroundHovered: '#373737',
+  menuItemBackgroundPressed: 'rgba(128,128,128,.6)',
+  primaryButtonBackground: 'rgb(40, 153, 245)',
+  primaryButtonBackgroundDisabled: 'rgb(50, 49, 48)',
+  primaryButtonBackgroundHovered: 'rgb(58, 160, 243)',
+  primaryButtonBackgroundPressed: 'rgb(108, 184, 246)',
+  primaryButtonBorder: '#4894fe',
+  primaryButtonText: 'rgb(27, 26, 25)',
+  primaryButtonTextDisabled: 'rgb(121, 119, 117)',
+  primaryButtonTextHovered: 'rgb(27, 26, 25)',
   primaryButtonTextPressed: DarkSemanticColors.primaryButton.pressed.text,
+  textFieldBorder: '#8A8886',
+  textFieldBorderHover: '#323130',
+  textFieldBorderActiveFocus: '#0078D4',
+  thumbDisabled: 'rgb(50, 49, 48)',
+  thumbNotCheck: 'rgb(161, 159, 157)',
+  thumbDisabledNotChecked: 'rgb(72, 70, 68)',
+  thumbEnabledChecked: 'rgb(27, 26, 25)',
   variantBorder: CommonSemanticColors.dividers.lineSeparator,
   // extended
   controlAccent: DarkSemanticColors.controlOutlines.accent,
   controlOutline: DarkSemanticColors.controlOutlines.rest,
-  controlOutlineDisabled: DarkSemanticColors.controlOutlines.disabled,
-  controlOutlineHovered: DarkSemanticColors.controlOutlines.hover,
+  controlOutlineDisabled: 'rgb(72, 70, 68)',
```

Comment:
Would it make more sense to update the DarkSemanticColors.controlOutlines type with these values?

Reply:
Going to chat with Rick about this.

Reply:
David, all values have been updated per your recommendation.